docs/application/design/wearable/ui-components/pages.md

```diff
@@ -9,35 +9,35 @@ Pages are where all the content of an app is presented. At any moment, an app sh
 
 -   **Header**
 
-    The header includes the page title, which should be text-only without icons. Titles that are longer than the space provided slide in and fade out at the right edge of the screen. For a list page, you can include action buttons in the header.
+    The header includes the page title, which should be text-only without icons. Titles that are longer than the space provided slide in and fade out at the right edge of the screen. For a list page, you can also include action buttons in the header.
 
 -   **Body**
 
-    The body of a page can contain text or image content.
+    The body of a page contains content including the main content, images, buttons, or text.
 
     -   **Text content**
 
-        Text content should have margins on either side. Without these, it’s hard to read the text on a round display. When the text doesn't fit on one screen, the page scrolls vertically so users can see the full text. Refer to [Scroll bar](scroll-bars.md) for more details.
+        Text content should have margins on either side. Without these, it's hard to read the text on a round display. When the text doesn't fit on one screen, the page scrolls vertically so users can see the full text. Refer to [Scroll bar](scroll-bar.md) for more details.
```

Comment:
It looks like `scroll-bars.md` is correct. Would you pleaes check?

Reply:
done.